rebasehelper/cli.py

```diff
@@ -148,15 +159,103 @@ def __getattr__(self, name):
 class CliHelper(object):
 
     @staticmethod
+    def _check_results_dir(results_dir):
+        if os.path.exists(results_dir):
+            logger.warning("Results directory '%s' exists, removing it", os.path.basename(results_dir))
+            shutil.rmtree(results_dir)
+        os.makedirs(results_dir)
+        os.makedirs(os.path.join(results_dir, settings.REBASE_HELPER_LOGS))
+        os.makedirs(os.path.join(results_dir, 'old'))
+        os.makedirs(os.path.join(results_dir, 'new'))
+
+    @staticmethod
+    def _add_debug_log_file(results_dir):
+        debug_log_file = os.path.join(results_dir, settings.REBASE_HELPER_DEBUG_LOG)
+        try:
+            LoggerHelper.add_file_handler(logger,
+                                          debug_log_file,
+                                          logging.Formatter("%(asctime)s %(levelname)s\t%(filename)s"
+                                                            ":%(lineno)s %(funcName)s: %(message)s"),
+                                          logging.DEBUG)
+        except (IOError, OSError):
+            logger.warning("Can not create debug log '%s'", debug_log_file)
+        else:
+            return debug_log_file
+
+    @staticmethod
+    def _add_report_log_file(results_dir):
+        report_log_file = os.path.join(results_dir, settings.REBASE_HELPER_REPORT_LOG)
+        try:
+            LoggerHelper.add_file_handler(logger_report,
+                                          report_log_file,
+                                          None,
+                                          logging.INFO)
+        except (IOError, OSError):
+            logger.warning("Can not create report log '%s'", report_log_file)
+        else:
+            return report_log_file
+
+    @staticmethod
+    def setup(arguments):
```

Comment:
arguments should default to ```None```. Since ArgumentParser.parse() can be called without any argument or with None - https://docs.python.org/2/library/argparse.html#the-parse-args-method